parachain/pallets/inbound-queue/src/envelope.rs

```diff
@@ -30,10 +30,10 @@ pub struct Envelope {
 #[derive(Copy, Clone, RuntimeDebug)]
 pub struct EnvelopeDecodeError;
 
-impl TryFrom<Log> for Envelope {
+impl TryFrom<&Log> for Envelope {
```

Comment:
This change seems unnecessary and unrelated :)

Reply:
Or we need an extra clone here. https://github.com/Snowfork/snowbridge/blob/1033090342e61c1d5a86a893187794c971e553f1/parachain/pallets/inbound-queue/src/lib.rs#L236

Reply:
Ok, lets keep your change them